### PR TITLE
[OBMIN-62] Introduced updated Location model with UA and EN locations names in it

### PR DESCRIPTION
**Pull request by rpkyrych**
The new Location model will contain Area, District, and City fields for EN and UA supported languages.

### CODE REVIEW

**Review by Wolshebnik — APPROVED** [review]

File: src/test/java/space/obminyashka/items_exchange/i18n/InternationalizationTest.java [hunk 1]
```diff
@@ -48,15 +48,15 @@ void testDefaultLocalizationWithoutAccept_LanguageHeader() throws Exception {
     }
 
     @ParameterizedTest
-    @ValueSource(strings = {"ru", "ru-RU"})
-    void testRussianLocalizationWithAccept_LanguageHeaderIsRu(String locale) throws Exception {
+    @ValueSource(strings = {"ua", "ua-UA"})
+    void testUkrainianLocalizationWithAccept_LanguageHeaderIsRu(String locale) throws Exception {
         MvcResult mvcResult = mockMvc.perform(get(ApiKey.USER_MY_INFO)
```
Comment (Jack11M) [inline src/test/java/space/obminyashka/items_exchange/i18n/InternationalizationTest.java:53]: Please change the end of the test's name to isUa

File: src/test/java/space/obminyashka/items_exchange/service/LocationServiceTest.java [hunk 2]
```diff
@@ -120,19 +125,19 @@ void existsById_shouldReturnTrueIfLocationsWithGivenIdExists() {
...
-        when(locationRepository.saveAndFlush(any())).thenReturn(location);
-
-        LocationDto updatedLocationDto = locationService.update(locationDto);
-        verify(locationRepository, times(1)).saveAndFlush(locationCaptor.capture());
-        assertAll("Checking objects' data equal",
-                () -> assertEquals(locationCaptor.getValue().getId(), updatedLocationDto.getId()),
-                () -> assertEquals(locationCaptor.getValue().getCity(), updatedLocationDto.getCity()),
-                () -> assertEquals(locationCaptor.getValue().getDistrict(), updatedLocationDto.getDistrict()));
-    }
+//    @Test
```
Comment (Jack11M) [inline src/test/java/space/obminyashka/items_exchange/service/LocationServiceTest.java:128]: Remove the commented code here please

**Review by Jack11M — APPROVED** [review]

File: src/test/java/space/obminyashka/items_exchange/i18n/InternationalizationTest.java (hunk 1 above)
Comment (rpkyrych) [reply src/test/java/space/obminyashka/items_exchange/i18n/InternationalizationTest.java:53]: Nice catch!

File: src/test/java/space/obminyashka/items_exchange/service/LocationServiceTest.java (hunk 2 above)
Comment (rpkyrych) [reply src/test/java/space/obminyashka/items_exchange/service/LocationServiceTest.java]: Updated the test with new data.

**Pull request merged**